src/rpdk/core/contract/suite/handler_create.py

```diff
@@ -73,5 +73,4 @@ def contract_create_read_success(created_resource, resource_client):
 @pytest.mark.list
 def contract_create_list_success(created_resource, resource_client):
     created_model, _request = created_resource
-    models = test_list_success(resource_client, created_model)
-    assert created_model in models
+    assert test_list_success(resource_client, created_model)
```

Comment:
we already enforce the primary identifier to be in the created_model. i think checking here again is redundant, as the created_model will already have a primary identifier. if thats the case, just checking that the model is in the list of models is sufficient.

Reply:
ah forgot list only returns identifiers